packages/lesswrong/server/resolvers/forumEventResolvers.ts

```diff
@@ -47,38 +47,17 @@ addGraphQLResolvers({
       })
       return true
     },
-    AddForumEventSticker: async (
-      _root: void,
-      {forumEventId, x, y, theta}: {
-        forumEventId: string,
-        x: number,
-        y: number,
-        theta: number,
-      },
-      {currentUser, repos}: ResolverContext,
-    ) => {
-      if (!currentUser) {
-        throw new Error("Permission denied");
-      }
-
-      await repos.forumEvents.addSticker(forumEventId, currentUser._id, {x, y, theta})
-      captureEvent("addForumEventSticker", {
-        forumEventId,
-        userId: currentUser._id,
-        x, y, theta
-      })
-      return true
-    },
     RemoveForumEventSticker: async (
       _root: void,
-      {forumEventId}: {forumEventId: string},
+      {forumEventId, stickerId}: {forumEventId: string, stickerId: string},
       {currentUser, repos}: ResolverContext,
     ) => {
       if (!currentUser) {
         throw new Error("Permission denied");
       }
 
-      await repos.forumEvents.removeSticker(forumEventId, currentUser._id)
+      await repos.forumEvents.removeSticker({ forumEventId, stickerId, userId: currentUser._id })
```

Comment:
I feel like removing the sticker should also delete the associated comment (at least, I was surprised to see that my comment was still on the post)

Reply:
I've changed it to delete both. It's a bit of a hack but I've also made it copy the comment to your clipboard so you can un-delete it (I imagine some people might be annoyed to lose long comments if they clicked accidentally)
![Screenshot 2025-01-17 at 17 39 05](https://github.com/user-attachments/assets/7f7a7f94-771e-4645-a6d2-21b86fa23f48)

Reply:
ooo nice